.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish GitHub Package
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install nvim binary
+        uses: rhysd/action-setup-vim@v1
+        if: steps.cache-nvim.outputs.cache-hit != 'true' || steps.cache-nvim-windows.outputs.cache-hit != 'true'
+        with:
+          neovim: true
+          version: nightly
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1.8.1
+        with:
+          dotnet-version: 5.0.x
+
+      - name: Install dependencies
+        run: dotnet restore
+
+      - name: Build
+        run: dotnet build --no-restore
+
+      - name: Test
+        run: dotnet test --no-build --verbosity normal test/NvimClient.Test/NvimClient.Test.csproj
+
+      - name: Publish
+        run: |
+          dotnet nuget add source --username USERNAME --password ${{ secrets.GITHUB_TOKEN }} --store-password-in-clear-text --name github "https://nuget.pkg.github.com/neovim/index.json"
```

Comment:
any idea why `--store-password-in-clear-text` is needed?

Reply:
Not 100% sure, but it's the command they say to use here: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#authenticating-with-github_token-in-github-actions